pagopa/io-app
# chore(Firma con IO): [SFEQS-1706] Refactor ErrorComponent to use pictograms

## Short description
This PR refactors the FCI `ErrorComponent`to use pictograms instead of PNGs. 

## List of changes proposed in this pull request
- ts/features/fci/components/ErrorComponent.tsx: Changes the props to accept a pictogram name and export them for tests. Also adds the pictogram component rendering.
- ts/features/fci/components/GenericErrorComponent.tsx: Replaces PNGs with the correspondent pictograms;
- ts/features/fci/components/SuccessComponent.tsx: Replaces PNGs with the correspondent pictograms;
- ts/features/fci/screens/FciRouterScreen.tsx: Replaces PNGs with the correspondent pictograms;
- ts/features/fci/components/__tests__/ErrorComponent.test.tsx: Updates tests to use the exported `ErrorComponent` props and pictograms.

## How to test
Start an io-dev-api-server instance with the following configuration and check for any changed pictogram:
```
...
fci: {
      waitForSignatureCount: 1,
      rejectedCount: 1,
      expiredCount: 1,
      expired90Count: 1,
      waitForQtspCount: 1,
      signedCount: 1,
      canceledCount: 1,
      noSignatureFieldsCount: 0,
      response: {
        getFciResponseCode: 200
      }
    },
...
```
